dev-ruby/rdoc/rdoc-6.1.2.ebuild

```diff
@@ -82,6 +82,7 @@ all_ruby_install() {
 				ruby_fakegem_binwrapper $bin /usr/bin/${bin}${version}
 				sed -i -e "1s/env ruby/ruby${version}/" \
 					"${ED}/usr/bin/${bin}${version}" || die
+				if use prefix; then hprefixify ${ED}/usr/bin/${bin}${version}; fi
```

Comment:
The argument to hprefixify needs to be quoted here, since ${ED} may contain spaces.